package/gluon-core/luasrc/usr/lib/lua/gluon/util.lua

```diff
@@ -185,9 +185,11 @@ end
 -- 5: mesh1
 -- 6: owe1
 -- 7: wan_radio1 (private WLAN); mesh VPN
+-- 8: client2
+-- 9: mesh2
+-- A: ibss2
```

Comment:
We don't support IBSS, should be owe2

I'm also pondering if we should change `generate_mac` to take 2 arguments - a radio index + an address offset 0..3, where each radio index would result in a different hash seed (and the non-WLAN callers of `generate_mac` would pass 0 or 1 as the radio index)

Reply:
I think this was going into the direction of where i did propose this years ago.

I have another thought - In addition to calling this by indexes which effectively translate to an offset in the assigned block of n (8?) addresses, how about we use named indexes?

This way, we can track all handed out addresses by name. This has the upside of allowing 3rd party community packages to obtain unique per-radio addresses.